### ISSUE TITLE
minitar

### ISSUE THREAD

**Issue by Fxrh** [nogweii/arson#9]
To get arson working, I had to change the line
" require 'facets/minitar' " to
" require 'archive/tar/minitar' ".

using arson 2.1.1, facets 2.7.0
(see http://facets.rubyforge.org/2009/08/22/release-2-7.html)

Very nice app btw, I really like it :-)

**Comment by Daenyth** [comment]
This is probably related to the ruby 1.9 update.

**Comment by brettflorio** [comment]
Yes, and that minitar.rb is now deprecated in facets.  Nice fix, @Fxrh.
